src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -210,6 +210,11 @@ public void setStorageClassName(String storageClassName) {
 	 */
 	private String serviceAnnotations = null;
 
+	/**
+	 * Pod annotations to set for the pod created for each deployment.
+	 */
+	private String podAnnotations;
```

Comment:
I think we need a doc update for this (and for service annotations as well). Created https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/issues/205